src/components/new-request/name-check/name-check.vue

```diff
@@ -0,0 +1,809 @@
+          <v-btn @click="back()" class="outlined" outlined>
+            Start Over
+          </v-btn>
+        </v-col>
+        <v-col>
+          <v-btn id="name-check-submit-btn" style="float: right;" @click="dialogCheck()">
+            Submit this Name for Review
+            <v-icon>mdi-chevron-right</v-icon>
+          </v-btn>
+        </v-col>
+      </v-row>
+    </v-container>
+  </v-container>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Watch } from 'vue-property-decorator'
+import { Action, Getter } from 'vuex-class'
+
+import { NameCheckIssuesDialog } from '@/components/dialogs'
+import MainContainer from '@/components/new-request/main-container.vue'
+import NameCheckConflicts from '@/components/new-request/name-check/name-check-conflicts.vue'
+import NameCheckTabContent from '@/components/new-request/name-check/name-check-tab-content.vue'
+import NameInput from '@/components/new-request/name-input.vue'
+import QuickSearchNames from '@/components/new-request/name-check/quick-search-names.vue'
+
+import { EntityType, NameCheckErrorType } from '@/enums'
+import { DialogOptionsI, NameCheckErrorI, NameCheckItemIF } from '@/interfaces'
+import { ActionBindingIF } from '@/interfaces/store-interfaces'
+import { baseItemsConflicts, baseItemsStructure } from './resources'
+
+@Component({
+  components: {
+    MainContainer,
+    NameCheckConflicts,
+    NameCheckIssuesDialog,
+    NameCheckTabContent,
+    NameInput,
+    QuickSearchNames
+  }
+})
+export default class NameCheck extends Vue {
+  @Getter getConflictsConditional!: Array<string>
+  @Getter getConflictsExact!: Array<string>
+  @Getter getConflictsRestricted!: Array<string>
+  @Getter getConflictsSimilar!: Array<string>
+  @Getter getDesignation!: string
+  @Getter getDesignationsCheckUse!: Array<string>
+  @Getter getDesignationsMismatched!: Array<string>
+  @Getter getDesignationsMisplaced!: Array<string>
+  @Getter getEntityTextFromValue!: string
+  @Getter getEntityTypeCd!: EntityType
+  @Getter getFullName!: string
+  @Getter getIsXproMras!: boolean
+  @Getter getLocationText!: string
+  @Getter getNameCheckErrors!: NameCheckErrorI
+  @Getter getRequestActionCd!: string
+  @Getter getSpecialCharacters!: Array<string>
+  @Getter isAnalyzeConflictsPending!: boolean
+  @Getter isAnalyzeDesignationPending!: boolean
+  @Getter isAnalyzeStructurePending!: boolean
+  @Getter isMissingDescriptive!: boolean
+  @Getter isMissingDesignation!: boolean
+  @Getter isMissingDistinctive!: boolean
+
+  @Action getNameAnalysis!: ActionBindingIF
+  @Action nameCheckClearError!: ActionBindingIF
+  @Action setActiveComponent!: ActionBindingIF
+  @Action setDesignation!: ActionBindingIF
+  @Action startAnalyzeName!: ActionBindingIF
+  @Action startQuickSearch!: ActionBindingIF
+
+  config = {
+    modules: {
+      toolbar: false
+    },
+    placeholder: '',
+    scrollingContainer: true
+  }
+
+  private checks: string = null
+  private expandHelpTxt: boolean = false
```

Comment:
Where the type can be inferred from its initial value, we should omit the type. This is because some linters will complain about "redundant type declaration".

Reply:
oh okay sounds good